.github/workflows/bin/generate_cron_workflows.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+from jinja2 import Environment, FileSystemLoader
+
+
+def validate_inputs(cli_args):
+    try:
+        epoch = cli_args[1]
+    except Exception:
+        raise ValueError('Missing required parameter. '
+                         'Epoch must be provided.')
+
+    return epoch
+
+
+if __name__ == '__main__':
+
+    def write_recipe(jinja):
+        epoch = validate_inputs(sys.argv)
+        for os in 'macos', 'ubuntu':
+            filename = 'cron-' + epoch + '-core-' + os + '-latest.yaml'
+            env = Environment(loader=FileSystemLoader('templates'),
+                              variable_start_string='{{{',
+                              variable_end_string='}}}')
+            template = env.get_template(jinja)
+            with open(filename, 'w') as outfile:
+                if os == 'macos':
+                    opsys = 'osx'
+                else:
+                    opsys = 'linux'
```

Comment:
I would like to see you consolidate this `if/else` statement into your outer `for` loop (`for os in ...`) above. You can make this go from being a reactive check ("oh wait, what os am I?") to being a prescriptive declaration ("ok, in this cycle of the loop, the os is this, and the opsys is this other thing").

My suggestion is to loop over tuples, like this:

```python
>>> for x, y in [(1, 2), (3, 4)]:
...     print(x, y)
...
1 2
3 4
```